### PR TITLE
[Chore] Minor cleanup

### PR DESCRIPTION
**Pull request by kittinunf**

### CODE REVIEW

**Comment by worker8-1** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          9 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/drawable/ic_logo_foreground.xml#L15">app/src/main/res/drawable/ic_logo_foreground.xml#L15</a> - Attribute <code>endX</code> is only used in API level 24 and higher (current min is 21)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/drawable/ic_logo_foreground.xml#L16">app/src/main/res/drawable/ic_logo_foreground.xml#L16</a> - Attribute <code>endY</code> is only used in API level 24 and higher (current min is 21)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/drawable/ic_logo_foreground.xml#L17">app/src/main/res/drawable/ic_logo_foreground.xml#L17</a> - Attribute <code>startX</code> is only used in API level 24 and higher (current min is 21)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/drawable/ic_logo_foreground.xml#L18">app/src/main/res/drawable/ic_logo_foreground.xml#L18</a> - Attribute <code>startY</code> is only used in API level 24 and higher (current min is 21)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/drawable/ic_logo_foreground.xml#L22">app/src/main/res/drawable/ic_logo_foreground.xml#L22</a> - Attribute <code>offset</code> is only used in API level 24 and higher (current min is 21)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/drawable/ic_logo_foreground.xml#L25">app/src/main/res/drawable/ic_logo_foreground.xml#L25</a> - Attribute <code>offset</code> is only used in API level 24 and higher (current min is 21)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/drawable/ic_outline_bug.xml#L8">app/src/main/res/drawable/ic_outline_bug.xml#L8</a> - Very long vector path (852 characters), which is bad for performance. Considering reducing precision, removing minor details or rasterizing vector.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/layout/comment_header.xml#L7">app/src/main/res/layout/comment_header.xml#L7</a> - Possible overdraw: Root element paints background <code>?pixelBackgroundColor</code> with a theme that also paints a background (inferred theme is <code>@style/AppTheme</code>)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/worker8/Pixels/blob/26ab8177aa5f20b01cce8b97fbe2e3509ccf21c6/app/src/main/res/mipmap-hdpi/ic_launcher_round.png#L0">app/src/main/res/mipmap-hdpi/ic_launcher_round.png#L0</a> - Launcher icon used as round icon did not have a circular shape</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/src/main/java/beepbeep/pixelsforredditx/common/HeaderFooterPagedListAdapter.kt [hunk 1]
```diff
@@ -9,7 +9,7 @@ abstract class HeaderFooterPagedListAdapter<T>(diffCallback: DiffUtil.ItemCallba
     open var hasHeader: Boolean = false
     open var hasFooter: Boolean = false
 
-    fun HeaderFooterAdapter(hasHeader: Boolean, hasFooter: Boolean) {
+    fun headerFooterAdapter(hasHeader: Boolean, hasFooter: Boolean) {
```
Comment (worker8) [inline app/src/main/java/beepbeep/pixelsforredditx/common/HeaderFooterPagedListAdapter.kt]: I think this file can actually be removed

Comment (kittinunf) [reply app/src/main/java/beepbeep/pixelsforredditx/common/HeaderFooterPagedListAdapter.kt]: I will remove it then!

**Comment by kittinunf** [comment]
Thanks for the review!!! I will merge as soon as I got 🍏 from CI

File: app/build.gradle [hunk 2]
```diff
@@ -48,11 +48,13 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-extensions:$Versions.lifecycleEx"
     implementation "androidx.paging:paging-runtime:$Versions.paging"
     implementation "androidx.recyclerview:recyclerview:$Versions.androidX"
-    implementation 'com.google.android.material:material:1.1.0-alpha03'
+    implementation "com.google.android.material:material:$Versions.androidX"
```
Comment (worker8-1) [inline app/build.gradle]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/814cae669abc38d21778fc91279b72bfb75a1599/app/build.gradle#L51"></span>A newer version of com.google.android.material:material than 1.1.0-alpha02 is available: 1.1.0-alpha03</td>
    </tr>
  </tbody>
</table>

**Comment by kittinunf** [comment]
Merging this if you are okay with this! @worker8

**Comment by worker8** [comment]
> image

nice commit message 👍 
and thanks!

**Pull request merged**